docs/api/en/core/BufferGeometry.html

```diff
@@ -221,8 +221,7 @@ <h3>[method:BufferAttribute deleteAttribute]( [param:String name] )</h3>
 
 		<h3>[method:undefined dispose]()</h3>
 		<p>
-		When a BufferGeometry is rendered, it allocates WebGL resources which can only be disposed of by calling this method.
-		For more details read: [link:https://threejs.org/docs/index.html#manual/en/introduction/How-to-dispose-of-objects How to dispose of objects].
+		Frees the GPU-related resources allocated by this instance. Call this method whenever this instance is no longer used in your app.
```

Comment:
I believe it will be helpful to state that this is relevant only after the geometry is rendered, so intermediate non-rendered geometries don't need to be disposed of. For instance: [the internal geometry in TessellateModifier](https://github.com/erasta/three.js/blob/dev/examples/jsm/modifiers/TessellateModifier.js#L26)
We had some discussion on the subject here https://github.com/mrdoob/three.js/pull/24616 and there https://github.com/mrdoob/three.js/pull/24607